components/bookstore.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BoOkHuB</title>
+
+    <!-- Icon link -->
+    <link rel="icon" href="../assest/icon-logo.png">
+
+    <link rel="stylesheet" href="https://unpkg.com/swiper@7/swiper-bundle.min.css" />
+
+    <!-- font awesome cdn link  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+
+    <script src="https://kit.fontawesome.com/a3941b6a24.js" crossorigin="anonymous"></script>
+
+    <!-- custom css file link  -->
+    <link rel="stylesheet" href="../css/component_css/bookstore.css">
+
+</head>
+
+<body>
+
+    <!-- header section starts  -->
+    <header class="header">
+        <div class="header-bookhub">
+            <a href="#" class="logo"><img src="../assest/icon-logo.png" alt="icon-logo">BoOkHuB</a>
+            <form action="" class="search-form">
+                <input type="search" name="" placeholder="Find your next favorite book" class="input-bookhub"
+                    id="search-box">
+                <label for="search-box" class="fas fa-search search-icon"></label>
+            </form>
+            <div class="icons">
+                <div id="search-btn" class="fas fa-search header-icons"></div>
+                <a href="#" class="fas fa-heart header-icons">
+                    <div class="badge-icon">1</div>
+                </a>
+                <a href="#" class="fas fa-shopping-cart header-icons">
+                    <div class="badge-icon">1</div>
+                </a>
+                <button id="login-btn" class="login-btn-bookhub">Login</button>
+                <button id="signUp-btn" class="signUp-btn-bookhub">Sign up</button>
+            </div>
+        </div>
+    </header>
+    <!-- header section ends -->
+
+    <!-- Filter section -->
+    <main class="bookhub-container">
+        <section class="filter-drawer">
+            <div class="filter-header">
+                <h3>Filters</h3>
+                <button class="btn">Clear</button>
+            </div>
+            <span class="hr_line-book"></span>
+            <div class="filter-category filter">
+                <h4>Category</h4>
+                <label for="fiction">
+                    <input type="checkbox" id="fiction" value="1">Fiction
```

Comment:
give some space between input and labels

Reply:
okay, done